### ISSUE TITLE
[monit] allow the disabling of the monit web server

### ISSUE THREAD

**Issue by odyssey4me** [rcbops/chef-cookbooks#609]
The monit web server serves little purpose unless it's specifically used in an environment. We believe that it should be disabled by default and have an attribute which allows the user to enable it if they choose to do so.

Leaving it enabled by default simply increases the attack surface for someone looking to cause trouble.

Resolved by https://github.com/rcbops-cookbooks/monit/pull/29

**Comment by jcannava** [comment]
@odyssey4me looks like this isn't passing jenkins because when it disables the http server monit returns 12:13:35 monit: Cannot connect to the monit daemon. Did you start it with http support?

**Comment by odyssey4me** [comment]
@jcannava Odd, we're running it with this pull in place without any errors at all and the recipe changes the behaviour so that it doesn't even try to start the http server. Does jenkins somehow expect it to be running?

**Comment by claco** [comment]
@odyssey4me @jcannava So, when you disable the httpd, monit status from the command line no longer works since it uses the httpd server.  The attribute default is set to false.

How about instead we set it to `true` (enabled), but se the bind host to `127.0.0.1`?

Or, to maintain past behavior, we simply set it to `true` and leave bind host as is, but with the patch now, people have the option to disable it, or bind down to 127.0.0.1.
